bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -298,15 +317,20 @@ func decodeInt(b []byte) (int64, int) {
 	bp := make([]byte, 8)
```

Comment:
I'll change this if you would like.

If you're interested:
Go's escape analysis is interesting here, as one would think it could stack allocate that, though the present form is less code with the equivalent result from the perspective of the runtime: 1 heap allocation, and 3 slice creations, with the difference being the slice being made during `bp[:]` instead of `make([]byte, 8)`.

The compiler doesn't stack allocate `bp` as an array because the slice that would be made before the `bytes.NewBuffer` call, `bp[:]`, and passed into the call, implicitly holds a pointer to it, and then the compiler doesn't try to make assumptions about passed pointers afterwards.

Here's the result of running `go build -gcflags '-m -m'` on the two versions of the file:
```
bindings/go/src/fdb/tuple/tuple.go:317:12: make([]byte, 8) escapes to heap
bindings/go/src/fdb/tuple/tuple.go:317:12:      from bp (assigned) at bindings/go/src/fdb/tuple/tuple.go:317:5
bindings/go/src/fdb/tuple/tuple.go:317:12:      from bytes.buf·2 (assign-pair) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:317:12:      from &bytes.Buffer literal (struct literal element) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:317:12:      from &bytes.Buffer literal (pointer literal) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:317:12:      from ~R0 (assign-pair) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:317:12:      from bytes.NewBuffer(bp) (interface-converted) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:317:12:      from bytes.NewBuffer(bp) (passed to call[argument escapes]) at bindings/go/src/fdb/tuple/tuple.go:321:13
```
```
bindings/go/src/fdb/tuple/tuple.go:321:32: bp escapes to heap
bindings/go/src/fdb/tuple/tuple.go:321:32:      from bytes.buf·2 (assign-pair) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:321:32:      from &bytes.Buffer literal (struct literal element) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:321:32:      from &bytes.Buffer literal (pointer literal) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:321:32:      from ~R0 (assign-pair) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:321:32:      from bytes.NewBuffer(bp[:]) (interface-converted) at bindings/go/src/fdb/tuple/tuple.go:321:29
bindings/go/src/fdb/tuple/tuple.go:321:32:      from bytes.NewBuffer(bp[:]) (passed to call[argument escapes]) at bindings/go/src/fdb/tuple/tuple.go:321:13
```

Reply:
Okay, I guess it's fine either way.